client/a8c-for-agencies/sections/partner-directory/agency-details/hooks/use-submit-form.ts

```diff
@@ -23,12 +36,29 @@ export default function useSubmitForm( { formData, onSubmitSuccess, onSubmitErro
 		},
 	} );
 
-	const onSubmit = useCallback( () => {
+	const onSubmit = useCallback( async () => {
+		if ( formData.logoUrl.startsWith( 'blob:' ) ) {
+			setIsUploadingImage( true );
+			// This mean we need to upload another image.
+			const file = await getImageFile( agencyId ?? 0, formData.logoUrl );
+
+			const media = await addMedia( [ file ], {
+				ID: 234537984, // https://a8cforagenciesportfolio.wordpress.com/ is the blog where we will dump profile images so it can be publicly access.
+				options: {
+					allowed_file_types: [ 'jpg', 'jpeg', 'png', 'gif' ],
```

Comment:
We don't allow `gif` images. Filipe said that to avoid logo with animations.

Reply:
Thanks, this has been addressed. 👍

Reply:
Yes, please. No GIF in order to avoid agencies uploading animated images and get unfair visibility on the PDs.